content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -50,4 +50,12 @@ The [data set](data.csv) consists of health and demographic data for the period
 
 9. How is life expectancy related to wealth across different regions? How is wealth related to fertility across different regions? Do these relationships hold for African countries?
 
-10. Create appropriate graphs to visually represent the relationship between literacy and life expectancy by region, and then for African countries. What can be concluded from the graphs? How confident can we be in the relationships represented here?
\ No newline at end of file
+10. Create appropriate graphs to visually represent the relationship between literacy and life expectancy by region, and then for African countries. What can be concluded from the graphs? How confident can we be in the relationships represented here?
+
+
+### Instructions for reviewers
+
+- Ensure that correct methods were used to check for any missing values in the data, also check how they dealt with missing values and if they gave a valid reason(s) for choosing that method.
+- Ensure that they checked if the data has any other problems and fixed the problems if they detected any problems.
```

Comment:
"Ensure that they checked if the data has any other problems and resolved them if so."